x-pack/plugins/observability_solution/infra/public/pages/metrics/inventory_view/components/waffle/metric_control/mode_switcher.tsx

```diff
@@ -5,98 +5,123 @@
  * 2.0.
  */
 
-import { EuiFlexGroup, EuiFlexItem, EuiButtonEmpty, EuiButton } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiButtonEmpty, EuiButton, EuiToolTip } from '@elastic/eui';
```

Comment:
We can include `useEuiTheme` here as well and remove line 13
```suggestion
import { EuiFlexGroup, EuiFlexItem, EuiButtonEmpty, EuiButton, EuiToolTip, useEuiTheme } from '@elastic/eui';
```